src/Nethermind/Nethermind.Synchronization/LesSync/ChtDecoder.cs

```diff
@@ -53,5 +53,21 @@ private int GetContentLength((Keccak?, UInt256) item, RlpBehaviors rlpBehaviors)
             (Keccak? hash, UInt256 totalDifficulty) = item;
             return Rlp.LengthOf(hash) + Rlp.LengthOf(totalDifficulty);
         }
+
+        public (Keccak?, UInt256) Decode(Span<byte> span) => Decode(new Rlp.ValueDecoderContext(span));
```

Comment:
Is this new? Where is this used at?

Reply:
This is old code for LES protocol should be removed.